recipes/conda-env/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "2.4.5" %}
+
+package:
+  name: conda-env
+  version: {{ version }}
+
+source:
+  fn: conda-env-{{ version }}.tar.gz
+  url: https://github.com/conda/conda-env/archive/v{{ version }}.tar.gz
+  sha256: 5ce2dd1b023a84ee76d19d508e519c1ca84010a60744976aa3caa42ba7ab38f9
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
```

Comment:
I don't know. It doesn't seem to be needed currently. Though I think `conda` and `conda-env` are setup that way. @mcg1969 and I had this conversation in the previous PR ( https://github.com/conda-forge/staged-recipes/pull/657 ).